geo/geofence.go

```diff
@@ -269,3 +277,47 @@ func MatchGeofences(featureCollection *geojson.FeatureCollection, lat, lon float
 
 	return
 }
+
+func (fence *Geofence) toFeature() *geojson.Feature {
+	var ring orb.Ring
+	for _, loc := range fence.Fence {
+		ring = append(ring, orb.Point{loc.Longitude, loc.Latitude})
+	}
+	polygon := orb.Polygon{ring}
+
+	return geojson.NewFeature(polygon)
+}
+
+func (fence *GeofenceApi) toGeofence() *Geofence {
+	var locations []Location
+	for _, loc := range fence.Fence {
+		locations = append(locations, loc.ToLocation())
+	}
```

Comment:
I've seen this pattern a lot and I have had plans to fix them. This is more efficient as:
```
locations := make([]Location, len(fence.Fence))
for i, loc := range fence.Fence {
	locations[i] = loc.ToLocation()
}
```

When you know the length up front, this is better, as it results in only 1 alloc... instead of possibly many.